docs/guides/cli/05_cli-basics.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: cli-basics
+title: Basic functionality
+---
+
+To get a full list of commands available in Ory CLI, run this command:
+
+```shell
+ory help
+```
+
+This prints the list of available commands with short descriptions.
+Additionally, each commands accepts the `--help` flag. Use it to get more
+information about the command, available command-specific flags and command
+modifiers. For example:
+
+```shell
+ory auth --help
+
+Create an or sign into your Ory Cloud account
+
+Usage:
+  ory auth [flags]
+  ory auth [command]
+
+Available Commands:
+  logout      Signs you out of your account on this computer.
+
+Flags:
+  -c, --config string   Path to the Ory Cloud configuration file.
+      --format string   Set the output format. One of table, json, and json-pretty. (default "default")
+  -h, --help            help for auth
+  -q, --quiet           Be quiet with output printing.
+  -y, --yes             Confirm all dialogues with yes.
+
+Use "ory auth [command] --help" for more information about a command.
+```
+
+## Create Ory Cloud Account & Project
+
+With Ory CLI, you can create a new Ory Cloud account and project from your
+terminal.
+
+Follow these steps:
+
+1. Run this command to start the wizard:
+
+```shell
+ory auth
+```
+
+2. If you haven't signed in with the CLI before, you're asked if you have an Ory
+   Cloud account.
+
+```shell
+Do you already have an Ory Console account you wish to use? [y/n]: n
+```
+
+Answer "no" to create a new account. In the process, you must provide the basic
+account information: email address, password, user name, and choose if you want
+to be informed about platform security updates.
+
+3. As a final step, accept the [Terms of Service](https://www.ory.sh/ptos).
+
+```shell
+I accept the Terms of Service https://www.ory.sh/ptos:  [y/n]: y
+```
+
+4. When the process is completed, the CLI displays the created project and user
+   details:
+
+```text
+You are now signed in as: tom@email.com
+ID			cae666dg-2ffa-491b-9912-6b090ec10e1c
+EMAIL			tom@email.com
+SELECTED_PROJECT	00000000-0000-0000-0000-00000000000
```

Comment:
It just looks bad in the output

<img width="756" alt="Bildschirmfoto 2022-03-30 um 11 48 45" src="https://user-images.githubusercontent.com/3372410/160803520-0fd9ff5d-5276-4cd2-a52e-1792f8279bb4.png">

align the items so that it looks nice :)

```suggestion
ID								cae666dg-2ffa-491b-9912-6b090ec10e1c
EMAIL							tom@email.com
SELECTED_PROJECT	00000000-0000-0000-0000-00000000000
```